documentation/text-utilities.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Roshan J" />
+    <meta name="description"
+        content="Gentle UI is a component library having pre built components with the motive of creating good UIs." />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Exo:wght@300;500;600;700&display=swap" />
+    <link rel="stylesheet" href="../gentle-imports.css" />
+    <link rel="stylesheet" href="documentation.css" />
+    <link rel="icon" href="https://img.icons8.com/stickers/100/000000/g.png " />
+    <title>Text Utilities | Gentle UI</title>
+</head>
+
+<body>
+    <div class="documentation-layout">
+        <div class="navbar p-4">
+            <div class="nav-left p-4">
+                <img src="https://img.icons8.com/stickers/60/000000/g.png" />
+                <h5><a href="../index.html" class="link-none">Gentle UI</a></h5>
+            </div>
+
+            <ul class="nav-right p-4">
+                <li>
+                    <a href="../index.html" class="link-none">HOME</a>
+                </li>
+                <li>
+                    <a href="https://github.com/dev-enforced/Gentle_UI" class="link-none">
+                        <i class="fab fa-github fa-2x"></i>
+                    </a>
+                </li>
+            </ul>
+        </div>
+        <div class="sidebar p-6">
```

Comment:
You can use `aside` tag for sidebar

Reply:
Yes will keep in mind while making the upcoming pages of the project